README.md

```diff
@@ -2,208 +2,394 @@
 
 ## Learning Goals
 
-- Explain in general terms what the execution context is.
-- Describe the difference between global- and function-scoped code.
-- Understand how block scoping affects variables declared with `let` and `const`.
+- Define scope as a programming concept.
+- Describe the three levels of scope in JavaScript.
+- Examine the exceptions to the scope rules.
```

Comment:
Consider updating "examine" to something that could be measured/assessed such as "Identify the exceptions...." or "Determine the exceptions...."